lib/service/forum/ForumMeldingenService.php

```diff
@@ -73,7 +90,21 @@ public function __construct(
 		$this->forumDelenMeldingRepository = $forumDelenMeldingRepository;
 		$this->lidInstellingenRepository = $lidInstellingenRepository;
 		$this->profielRepository = $profielRepository;
+		$this->webPushRepository = $webPushRepository;
 		$this->security = $security;
+
+		$auth = [
+			'VAPID' => [
+				'subject' => 'mailto:pubcie@csrdelft.nl',
+				'publicKey' => file_get_contents(
+					__DIR__ . '/../../../data/vapid_public_key.txt'
```

Comment:
De verwijzing naar deze keypair kan ook in de .env Kan je ook docs toevoegen over hoe dit werkt?

Reply:
Ja dat is prima!